lib/src/sv_common.c

```diff
@@ -563,6 +563,13 @@ parse_bu_info(const uint8_t *bu_data,
   return bu;
 }
 
+char *
+get_private_key_from_sv(signed_video_t *self)
```

Comment:
This function is only a wrapper of `get_private_key_from_sign_data()` so I think you should use that one directly in all the places.